### ISSUE TITLE
Implement model evaluation

### ISSUE THREAD

**Issue by LuXoN1999** [LuXoN1999/Segmentation-of-abdominal-organs-from-medical-images#27]
Display model predictions on validation dataset and overall generalization. 
## TODO
- after training process, add section for model evaluation(**should be independent from the training phase**)
- plot model predictions and compare it to original image and ground truth on validation set
- plot model predictions and compare it to original image on test set
- calculate IoU results over validation dataset

## DOCUMENTATION
https://matplotlib.org/stable/index.html
